packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,6 +1555,326 @@
           "nextNavigateTo": "P11"
         }
       ]
+    },
+    "P11": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "How many <defined-term>dwelling units</defined-term> are in this <defined-term>building</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "1-2",
+          "answerValue": "2"
+        },
+        {
+          "answerDisplayText": "Greater than 2",
+          "answerValue": "3"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
```

Comment:
Should nextNavigationLogic include
   "answerToCheck": "P11" ?

Reply:
I don't think so. This one only has fallback logic. It goes straight to P12. 

When Nick implements, the data will be stored so if p11 needs to be used it can be.